OpenRA.Mods.Common/Traits/Render/WithSpriteTurret.cs

```diff
@@ -71,6 +71,10 @@ public class WithSpriteTurret : ConditionalTrait<WithSpriteTurretInfo>, INotifyB
 		readonly Turreted t;
 		readonly Armament[] arms;
 
+		public bool UninterruptibleAnimationPlaying;
```

Comment:
Not sure. Maybe not right now, but there might be animation traits later that replace the sequence instead of using PlayCustomAnimation, yet still want to be uninterruptible.